### ISSUE TITLE
Polish CLIs MPC

### ISSUE THREAD

**Issue by AntoineRondelet** [clearmatics/zeth#90]
Just ran `mpc --help` on https://github.com/clearmatics/zeth/pull/73 and I have:
```
Usage:
  ./mpc [<options>] <command> <command-arguments> ...

  -h [ --help ]         This help
  -v [ --verbose ]      Verbose output

Commands:
  create-keypair  dummy-phase2  linear-combination  phase2-begin  phase2-contribute  phase2-verify-contribution  phase2-verify-transcript
```

1. I would follow the following formatting for the flags/options:
`-short-flags, --long-flags ARG       Explanation and usage`
2. All the commands are listed on the same line. It'd be great and clearer to list them in column and specify a brief description of the command.

So basically, I'd be great to have the following output for the `mpc --help`
```
Usage:
  ./mpc [<options>] <command> <command-arguments>

[mpc command description]

Options:
  -h, --help        Display this help and exit
  -v, --verbose      Verbose output

Commands:
  create-keypair                         Brief description of sub-command
  dummy-phase2                       Brief description of sub-command
  linear-combination                   Brief description of sub-command
  phase2-begin                          Brief description of sub-command
  phase2-contribute                   Brief description of sub-command
  phase2-verify-contribution       Brief description of sub-command
  phase2-verify-transcript           Brief description of sub-command
```

(I don't know how to align stuffs properly in markdown... all messages should consistently be aligned...)
Having such detailed help message is clearer. It would also (potentially) enable to rename several "long commands" with more succint names, like `linear-combination` -> `lin-comb`, `phase2-begin` -> `p2-begin` and so forth without losing clarity

**Comment by dtebbs** [comment]
Just FYI, I've added a small commit to #73 to address the part about the command list and description layout.

Regarding the format of the flags, that's controlled by `boost::program_options`, so I'm not sure how easy it is to change.

**Comment by AntoineRondelet** [comment]
Great thanks @dtebbs looks better!

> Regarding the format of the flags, that's controlled by boost::program_options, so I'm not sure how easy it is to change.

Ah yes indeed. Ok don't bother with that then!
This issue seems addressed (let's make sure to have similar outputs for other commands if applicable), but will close it when the PR is merged.

**Comment by AntoineRondelet** [comment]
Closing as this was fixed in #73
